_includes/concepts/assets.html

```diff
@@ -1,23 +1,34 @@
-<p class="lead">An asset in collins describes a thing, usually a piece of hardware or a configuration.</p>
-<p>
-  Assets themselves are very simple internally and only have a handful of meaningful attributes. An asset consists of a:
-</p>
+<p class="lead">An asset describes a piece of your infrastructure</p>
+<p>Assets are simple representations of a part of your infrastructure topology.
+They can be things like Routers and Switches, or Servers, Chassis, Configuration,
+Datacenter, or even user-defined types for your specific needs. Assets primarily consist of:</p>
 <dl>
-  <dt class="inlinecode">tag</dt>
-  <dd>A tag is a unique, immutable, alphanumeric sequence of characters associated with the asset. These are arbitrary but nearly all API calls for interacting with asset\
-    s will reference this tag.</dd>
+  <dt>Asset Tag</dt>
+  <dd>A tag is a unique, immutable, alphanumeric sequence of characters associated with the asset. These are arbitrary but nearly all API calls for interacting with assets will reference this tag. For servers, these are generally burned into the FRU/DMI and identify the asset. For physical assets, it is very useful to mark the asset tag on the outside of the device.</dd>
 
-  <dt class="inlinecode">status</dt>
-  <dd>Indicates the current lifecycle phase of the asset. More details in the next section.</dd>
+  <dt>Status</dt>
+  <dd>Indicates the current lifecycle phase of the asset. More details in <a href="#status & state">Status & State</a></dd>
 
-  <dt class="inlinecode">state</dt>
-  <dd>Indicates a phase (status) specific state of the asset. More details in the next section.</dd>
+  <dt>State</dt>
+  <dd>Indicates a phase (status) specific state of the asset. Operators can define arbitrary states with <a href="/api.html#state">the API</a>. More details in the next section.</dd>
 
-  <dt class="inlinecode">type</dt>
-  <dd>Current supported types include: Configuration, Data Center, Power Circuit, Power Strip, Rack, Router, Server Node and Switch. Of primary interest to most users wil\
-    l be server nodes and configurations.</dd>
+  <dt>Type</dt>
+  <dd>Describes the type of the asset. Of primary interest to most users will be server nodes and configurations.</dd>
+
+  <dt>Tags</dt>
```

Comment:
Most of the documentation refers to them as tags. I use synonyms in parens where possible to spread other terminology.